### ISSUE TITLE
Cocoapods 5.0.0 not upload to trunck yet

### ISSUE THREAD

**Issue by stonezhl** [tid-kijyun/Kanna#213]
### Description:
When trying to install kanna with cocoapods with 5.0.0 version, get the error no compatible version or name.

### Installation method:
- [ ] Carthage
- [x] CocoaPods
- [ ] Swift Package Manager
- [ ] Manually
- [ ] other: ()

### Kanna version (or commit hash):
5.0.0

### swift --version
5.0

### Xcode version (optional):
12.2

**Comment by tid-kijyun** [comment]
Would you please try the following command:

```
pod repo update
```

**Comment by stonezhl** [comment]
Thanks for your reply.
Actually, I met this issue because I have a pod spec depended on your repo.
`spec.dependency "Kanna"#, "~> 5.0.0"`
I called this command to verify
`pod lib lint` and get this error:
`- ERROR | [iOS] unknown: Encountered an unknown error (CocoaPods could not find compatible versions for pod "Kanna":`
And also I checked https://github.com/CocoaPods/Specs/tree/9543734012238c33ea018af5a4381459c13f0916/Specs/e/f/a/Kanna
Cannot find 5.0.0, maybe I misunderstand something.
